src/vs/base/browser/ui/splitview/paneview.ts

```diff
@@ -158,6 +159,11 @@ export abstract class Pane extends Disposable implements IView {
 		this.updateHeader();
 
 		if (expanded) {
+			if (!this._bodyRendered) {
+				this.renderBody(this.body);
+				this._bodyRendered = true;
```

Comment:
@sbatten We should set this to true inside `renderBody`, not outside. Especially because `renderBody` is `protected`.

Reply:
are you suggesting to move it out into `ViewPane` instead? `renderBody` is abstract in the `Pane`. Even if a subclass calls this, setting it to true inside `renderBody` won't prevent the subclass from calling it.

Reply:
I'm suggesting making it concrete and have every subclass call `super.renderBody()` in their `renderBody` implementation first thing.